HWrk_5/hw_26.py

```diff
@@ -0,0 +1,19 @@
+# Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+# *Пример:*
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+def degree (a, b):
+    i = 0
+    if b == 0:
+        return 1
+    elif b == 1:
+        return a
+    else:
+        i += 1
+        return a * degree(a, b - i)
```

Comment:
Переменная `i` не нужна, ведь в каждом вызове функции она сначала будет равна `0`, а позже `1`. Кроме того, блоки кода `else` тоже не нужны, так как `return` завершает работу функции
```suggestion
def degree (a, b):
    if b == 0:
        return 1
    if b == 1:
        return a
    return a * degree(a, b - 1)
```